css/teams/1/landing-page.css

```diff
@@ -1,2 +1,336 @@
 @import url(../../colors.css);
+@import url(../../typography.css);
 
+*, *::after, *::before { box-sizing:border-box; margin: 0; padding: 0; }
+
+h1,h2,h3,h4,h5,h6,p{
+    margin: 0;
+    padding: 0;
+    color: #101828;
```

Comment:
please use colors variable, this heading styles are common also, please include them in the main.css file, and review that is not repeated

Reply:
Same with the *, *::after, *::before { box-sizing:border-box; margin: 0; padding: 0; } I will recommend to make it common